GameData/RealismOverhaul/RO_RecommendedMods/TACLS/RO_TACLS_Tanks.cfg

```diff
@@ -130,10 +173,37 @@
 	@description = A simple CO2 scrubber using lithium hydroxide. Absorbs CO2, and releases water (which needs purification before use) and lithium carbonate (waste). Requires small electric charge to run air pump. Figure 0.75L of LithiumHydroxide/person/day. Rated for 3 people. Resources for 14days.
 	@MODULE[TacGenericConverter]
 	{
+		//conversionRate = 3.0	// # of people - Figures based on per/person
+		//inputResources = CarbonDioxide, 0.0058912100, ElectricCharge, 0.010, LithiumHydroxide, 0.0000085683
+		//outputResources = WasteWater, 0.003341591, true, Waste, 0.0000257297, true
 		@converterName = CO2 Scrubber
-		@conversionRate = 3.0	// # of people - Figures based on per/person
-		@inputResources = CarbonDioxide, 0.0058912100, ElectricCharge, 0.010, LithiumHydroxide, 0.0000085683
-		@outputResources = WasteWater, 0.003341591, true, Waste, 0.0000257297, true
+		@INPUT_RESOURCE,0
+		{
+			@ResourceName = ElectricCharge
+			@Ratio = 0.03
+		}
+		@INPUT_RESOURCE,1
+		{
+			@ResourceName = CarbonDioxide
+			@Ratio = 0.01767363
+		}
+		@INPUT_RESOURCE,2
+		{
+			@ResourceName = LithiumPeroxide
```

Comment:
It previously used LithiumHydroxide, while here it is changed to LithiumHydroxide.

I may still just merge this, since it will be an easy correction to do myself afterward.